src/sections/events/CreateEvent.js

```diff
@@ -4,16 +4,39 @@ import {
   TextField,
   Typography,
 } from "@mui/material";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { DateTimePicker } from "@mui/x-date-pickers";
 import { Stack } from "@mui/system";
 import { Icon } from "@iconify/react";
 import React, { useState } from "react";
 import { useRouter } from "next/router";
 import { PrimaryButton, SecondaryButton } from "@components/button";
 import { PATH_EVENTS } from "@routes/paths";
+import { createEvent } from "@redux/reducers/eventReducer";
+
+import library from "@utils/wallet";
+import { useDispatch, useSelector } from "react-redux";
 
 const CreateEvent = () => {
   const { push } = useRouter();
+  const dispatch = useDispatch();
+
+  let user = useSelector((state) => state.user);
+  user = user ? user : [];
+
+  const [startDateAndTimevalue, setStartDateAndTimeValue] = useState(
+    new Date("2014-08-18T21:11:54")
+  );
+  const [endDateAndTimevalue, setEndDateAndTimeValue] = useState(
+    new Date("2014-08-18T21:09:34")
+  );
+
+  const dummyContractAddress = Math.floor(1000 + Math.random() * 9000);
```

Comment:
please do not send dummyContractAddress.  figure out how to allow null in database